docs/source/CHANGELOG.rst

```diff
@@ -2,12 +2,21 @@
 Changelog
 =========
 
-1.1.1 - 2021-06-24
+1.2.0 - 2021-07-12
 ------------------
 
 Changed
+^^^^^
```

Comment:
I wonder if this will show an error message or not. Usually the number of `^` symbol cannot be less than the number of upper line characters.